cli/Cargo.toml

```diff
@@ -17,22 +17,23 @@ crate-type = ["cdylib", "rlib"]
 log       = { version = "0.4" }
 structopt = { version = "0.3", optional = true }
 # darwinia-network service
-service = { package = "darwinia-service", default-features = false, path = "../node/service" }
+darwinia-rpc = { path = "../rpc" }
```

Comment:
Delete this: See https://github.com/darwinia-network/darwinia-common/pull/1007/files#diff-9d8fd1afc09703e8bfc340998f6b2b728c5f2446a597e3d7f0049d10a7420bfd

```toml
trie-memory-tracker = ["sp-trie/memory-tracker"]
```